test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -44,6 +44,9 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
+import org.slf4j.Logger;
```

Comment:
this change was to make sure I was dropping the correct messages.  I think trunk got a similar version already